src/cmds/qstat.c

```diff
@@ -2012,6 +2100,9 @@ main(int argc, char **argv, char **envp) /* qstat */
 					errflg++;
 				}
 				break;
+			case 'E':
+				E_opt = 1;
```

Comment:
To maintain the similarity with other options, will leave it as it is. Will document this.